tests/cover/test_sampled_from.py

```diff
@@ -17,11 +17,20 @@
 
 from __future__ import division, print_function, absolute_import
 
+import pytest
+
 from hypothesis import given, settings
-from hypothesis.strategies import sampled_from
+from hypothesis.errors import HypothesisDeprecationWarning
+from hypothesis.strategies import sets, integers, sampled_from
 
 
 @given(sampled_from((1, 2)))
 @settings(min_satisfying_examples=10)
 def test_can_handle_sampling_from_fewer_than_min_satisfying(v):
     pass
+
+
+@given(sets(integers(), min_size=2))
+def test_cant_sample_sets(s):
+    with pytest.raises(HypothesisDeprecationWarning):
+        sampled_from(s).example()
```

Comment:
Let’s add a test that we can still sample from sets of size 1 without getting a deprecation warning. (A bit boring, but it is a distinct code path.)